exercises/practice/sum-of-multiples/tests/sum_of_multiples.cairo

```diff
@@ -0,0 +1,144 @@
+use sum_of_multiples::{sum_of_multiples as test_sum_of_multiples};
+
+#[test]
+fn no_multiples_within_limit() {
+    let mut factors: Array<u32> = array![3, 5];
```

Comment:
```suggestion
    let factors: Array<u32> = array![3, 5];
```
I don’t believe `factors` still needs to be mutable for any of the test cases. I believe that’s the last edit from me so Nenad the maintainer can take it from here.